tools/plugins/com.liferay.ide.gradle.core/src/com/liferay/ide/gradle/core/ModuleFragmentProjectProvider.java

```diff
@@ -255,4 +260,26 @@ public IStatus validateProjectLocation( String projectName, IPath path )
         return retval;
     }
 
+    private IPath createTempProject( String projectName )
+    {
+        final IWorkspaceRoot root = ResourcesPlugin.getWorkspace().getRoot();
+        final IProgressMonitor progressMonitor = new NullProgressMonitor();
+        final IProject project = root.getProject( TEMP_DIRECTORY );
```

Comment:
We need to try to implement this without using a temp project.  Discuss with Terry other options.